lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -102,10 +102,10 @@ def define
             end
           when "winter_fuel_payment"
             if calculator.country_eligible_for_winter_fuel_payment?
-              if calculator.going_abroad && calculator.country == "ireland"
+              if calculator.country == "ireland"
                 question :is_british_or_irish?
               else
-                outcome :worked_in_eea_or_switzerland? # A7 already_abroad
+                question :worked_in_eea_or_switzerland? # A7 already_abroad
```

Comment:
Minor: Ideally this would be fixed up to the commit where we originally added `outcome`, as it's always been a `question`.